docs/Config_Reference.md

```diff
@@ -1835,6 +1835,42 @@ z_offset:
 #   See the "probe" section for more information on the parameters above.
 ```
 
+### [load_cell_probe]
+
+Load-cell based probe. One may define this section (instead of a probe
+section) to enable a load-cell based probe. See
+[Load-cell probe guide](LoadCellProbe.md) and
+[command reference](G-Codes.md#load_cell_probe) for further
+information. Some of the parameters are determined in a semi-automatic
+[calibration procedure](LoadCellProbe.md#setup-for-new-printer-models).
+
+```
+[load_cell_probe]
+#
+# General parameters:
+#
```

Comment:
It would be preferable to layout the config reference in the same format as the other config sections.  Each parameter should state what its default is, or explicitly state that it must be provided.  FWIW, I'm not sure it helps for the reference to state which values the user can be guided to configuring.